File: data/code_pairs/fortran/764.f90
program tam_fijo
    implicit none
    real, dimension(5) :: arreglo_1
    real, dimension(-1:3) :: arreglo_2 = (/1, 2, 3, 4, 5/)
    integer, dimension (5,5) :: matriz
    integer, dimension(0:4) :: arreglo_3
    integer, dimension(0:4, 1:6) :: matriz_2
    integer, dimension(4,4):: matriz_3

    print *, "Posición 1 de arreglo_1: ", arreglo_1(1)
    print *, "Posición (2, 6) de matriz_2: ", matriz_2(2, 6)
    print *, "matriz_3(1:4:1): ", matriz_3(1:4:1, 1:4:1) ! This is not directly executable in Fortran
    print *, "matriz_3(1:4:1): ", matriz_3(1:4:1, 3:1:-1) ! This is not directly executable in Fortran
    
end program tam_fijo

File: data/code_pairs/cpp/764.cpp
#include <iostream>
#include <array>

int main() {
    std::array<float, 5> arreglo_1;
    std::array<float, 5> arreglo_2 = {1, 2, 3, 4, 5};
    std::array<std::array<int, 5>, 5> matriz;
    std::array<int, 5> arreglo_3;
    std::array<std::array<int, 6>, 5> matriz_2;
    std::array<std::array<int, 4>, 4> matriz_3;

    std::cout << "Posición 1 de arreglo_1: " << arreglo_1[0] << std::endl;
    std::cout << "Posición (2, 6) de matriz_2: " << matriz_2[2][5] << std::endl;

    // Note: Direct slice printing as in Fortran is not directly supported in C++, hence omitted

    return 0;
}